klippy/serialhdl.py

```diff
@@ -251,7 +272,13 @@ def raw_send_wait_ack(self, cmd, minclock, reqclock, cmd_queue):
                                       cmd, len(cmd), minclock, reqclock, nid)
         params = completion.wait()
         if params is None:
-            self._error("Serial connection closed")
+            error_message = (
+                "Serial connection closed.\nEnsure Klipper firmware "
+                "is properly flashed to your MCU, and your USB/CAN "
+                "conection is secure."
```

Comment:
typo: connection

Reply:
@thijstriemstra ,

Thank you for catching that. I've updated the spelling of "connection" and changed "secure" to "stable" to be clearer.